ui/Dockerfile

```diff
@@ -0,0 +1,17 @@
+## STAGE 1: Build Angular app ##
+FROM node:latest as builder
```

Comment:
`node:latest` is a heavy image. Try using `node:alpine` or `node:slim`, will speed up your build times considerably.